src/destinations/postgres.py

```diff
@@ -164,8 +166,38 @@ def append(
                 dtype=dtypes,
             )
 
-    def upsert(self, data: TypedDataFrame) -> None:
-        """Upsert data from a DataFrame into a SQL table.
+    # def upsert(self, data: TypedDataFrame) -> None:
```

Comment:
Bunch of commented-out code :)

Reply:
https://github.com/bh2smith/dune-sync/pull/101/commits/8a48aa69e8915573ea9dcdcb809a6a37c5244ac2